packages/core/src/Site/index.js

```diff
@@ -816,6 +819,51 @@ class Site {
     }
   }
 
+  async reloadSiteConfig() {
+    const oldAddressablePages = this.addressablePages.slice();
+    const oldPagesSrc = oldAddressablePages.map(page => page.src);
+    await this.readSiteConfig();
+    this.collectAddressablePages();
+    // Get pages with edited attributes but with the same src
```

Comment:
could this be scoped to the `else` branch?

Reply:
Yup this would be better since `editedPages` is only used inside the `else` branch.